spec/firebase_app_distribution_action_spec.rb

```diff
@@ -27,6 +27,87 @@
     Faraday.default_connection = nil
   end
 
+  describe '#get_value_from_value_or_file' do
+    it 'should return the release notes string' do
+      expect(Fastlane::Actions::FirebaseAppDistributionAction.get_value_from_value_or_file("Hello World", "")).to eq("Hello World")
+    end
+
+    it 'should return the release notes from file' do
+      expect(File).to receive(:open)
+        .with("release_notes_path")
+        .and_return(fake_binary)
+      expect(Fastlane::Actions::FirebaseAppDistributionAction.get_value_from_value_or_file("", "release_notes_path")).to eq("Hello World")
```

Comment:
This tests the case when the value is an empty string. Should you test the nil case as well?